convene-web/app/models/system_test_workspace.rb

```diff
@@ -36,12 +36,17 @@ def initialize(client_repository)
     end
 
     def find_or_create_workspace!
+      workspace = client.workspaces.find_or_create_by!(name: 'System Test Branded Domain')
+      workspace.update!(jitsi_meet_domain: 'convene-videobridge-zinc.zinc.coop',
+                        branded_domain: "#{ENV.fetch('HEROKU_APP_NAME', 'convene-pr-xxx')}.herokuapp.com",
```

Comment:
the `default_branded_domain` logic is getting complicated enough that we may want to pull it into it’s own function